pyemu/pst/pst_handler.py

```diff
@@ -2584,35 +2584,35 @@ def proportional_weights(self, fraction_stdev=1.0, wmax=100.0, leave_zero=True):
             new_weights.append(ow)
         self.observation_data.weight = new_weights
 
-    def calculate_pertubations(self):
+    def calculate_perturbations(self):
```

Comment:
This change will raise AttributeError if anyone tries to call `calculate_pertubations()`. If this is an issue, I can re-add the former method with a depreciation warning.